src/serverenvironment.cpp

```diff
@@ -582,6 +582,19 @@ void ServerEnvironment::addActiveBlockModifier(ActiveBlockModifier *abm)
 	m_abms.emplace_back(abm);
 }
 
+bool ServerEnvironment::replaceActiveBlockModifier(ActiveBlockModifier *abm)
+{
+	std::string name(abm->getName());
+	for (auto &&m_abm : m_abms) {
```

Comment:
Is this the move operator? Why?
```suggestion
	for (ActiveBlockModifier *&m_abm : m_abms) {
```